components/spin/index.tsx

```diff
@@ -115,18 +117,20 @@ export default class Spin extends React.Component<SpinProps, any> {
         [`${prefixCls}-blur`]: spinning,
       });
       return (
-        <Animate
-          {...divProps}
-          component="div"
-          className={`${prefixCls}-nested-loading`}
-          style={null}
-          transitionName="fade"
-        >
-          {spinning && <div key="loading">{spinElement}</div>}
-          <div className={containerClassName} key="container">
-            {this.props.children}
-          </div>
-        </Animate>
+        <div className={wrapperClassName}>
+          <Animate
+            {...divProps}
+            component="div"
+            className={`${prefixCls}-nested-loading`}
```

Comment:
Sorry, are you saying that insteat of creating a new _div_ I must modify the _className_ property of the `Animate` component to: `className={`${prefixCls}-nested-loading ${wrapperClassname}`}`?